mesa/src/personal_model/personal_model.js

```diff
@@ -40,15 +41,18 @@ class PersonalModel {
 
     __updateContext(){
         var _this = this;
-        // Calls __calculateContext after both activity and weather has been collected.
+        // Calls __calculateContext after time of day, activity and weather has been collected.
         var finished = _.after(2, _this.__calculateContext.bind(_this));
```

Comment:
```
var finished = _.after(2, _this.__calculateContext.bind(_this));
```

Instead of 2 it needs to be 3. The idea behind using this function is that __calculateContext is called automatically after all the data has been retrieved, because all these function are async we can't guarantee that they all happen at the same time. And we are waiting on:
1. time of day,
2. activity
3. weather

Reply:
Okay, changed to 3

Reply:
Nice!